src/test/java/com/plaid/client/integration/InstitutionsGetTest.java

```diff
@@ -70,6 +71,18 @@ public void testSuccessWithCountryCodes() throws Exception {
     assertEquals(3, response.body().getInstitutions().size());
   }
 
+  @Test
+  public void testSuccessWithOAuth() throws Exception {
+    Response<InstitutionsGetResponse> response =
+      client().service().institutionsGet(new InstitutionsGetRequest(3, 0).withCountryCodes(Arrays.asList("GB")).withOAuth()).execute();
```

Comment:
For the other tests in the file, can we also add an `assertFalse(institutions.get(i).getOAuth());` to make sure we aren't returning any `oauth` institutions  when we should not be?

Reply:
⚡️ daa9949